llumnix/queue/zmq_server.py

```diff
@@ -127,15 +128,15 @@ def get_nowait_batch(self, num_items):
             )
         return [self.queue.get_nowait() for _ in range(num_items)]
 
-    def _make_handler_coro(self, identity,
+    async def _make_handler_coro(self, identity,
                            message) -> Coroutine[Any, Any, Never]:
         request = cloudpickle.loads(message)
         if request == RPCUtilityRequest.IS_SERVER_READY:
-            return self._is_server_ready(identity)
+            return await self._is_server_ready(identity)
         if isinstance(request, RPCPutNoWaitQueueRequest):
-            return self._put_nowait(identity, request)
+            return await self._put_nowait(identity, request)
```

Comment:
_make_handler_coro function is used to return coroutine function, why await here?